### PR TITLE
Adding red to the header

### PR DESCRIPTION
**Pull request by jacobtie**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,11 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>My Page</title>
+  <style>
```
Comment (DylanHalstead) [inline index.html:8]: Don't we use Bootstrap in this course?

**Comment by Sm3ggl3s** [comment]
Red really makes it pop

**Review by DylanHalstead — CHANGES_REQUESTED** [review]

**Review by jadenPete — APPROVED** [review]
Beautiful.

File: index.html (hunk 1 above)
Comment (jacobtie) [reply index.html:8]: No, I'm a senior engineer

**Pull request merged**